public/app/features/alerting/unified/components/rules/RulesGroup.tsx

```diff
@@ -70,7 +71,7 @@ export const RulesGroup: FC<Props> = React.memo(({ group, namespace, expandAll }
     );
   } else if (rulesSource === GRAFANA_RULES_SOURCE_NAME) {
     if (folderUID) {
-      const baseUrl = `/dashboards/f/${folderUID}/${kbn.slugifyForUrl(namespace.name)}`;
+      const baseUrl = `${config.appSubUrl}/dashboards/f/${folderUID}/${kbn.slugifyForUrl(namespace.name)}`;
```

Comment:
This does look like the safest approach. :)

2 questions (non blockers):
* are we sure config.appSubUrl always has a leading / and no trailing / (e.g. "/suburl") or could that vary?
* is there any way to test this? I guess the better question is "is there a test that would have caught this bug?

Reply:
I have a similar question, is it possible to have undefined in `config.appSubrl` ???, because in this case we will have `undefined/dashboards'` as a result

Reply:
ok, no it's not possible , it's initialised with ''. So, 👍

Reply:
I'm not really sure if there is an easy way to test this. A one improvement that comes to my mind is to create a function that we always use when we create urls, then we could test that function and we would have unified way of building URLs

Reply:
this make sense @konrad147